packages/react/src/components/DashboardEditor/__snapshots__/DashboardEditor.story.storyshot

```diff
@@ -394,13 +395,14 @@ exports[`Storybook Snapshot tests and console checks Storyshots 2 - Watson IoT E
                         </div>
                         <div
                           className="bx--content-switcher iot--image-gallery-modal__content-switcher"
+                          data-testid="ComposedModal-content-switcher"
                           onChange={[Function]}
                           role="tablist"
                         >
                           <button
                             aria-pressed={null}
                             className="iot--icon-switch iot--icon-switch--large bx--btn bx--btn--secondary bx--btn--icon-only bx--tooltip__trigger bx--tooltip--a11y bx--tooltip--top bx--tooltip--align-center"
-                            data-testid="icon-switch"
```

Comment:
This looks like a change to a default, but the data-testid was only added to this button when this testId epic was started, so there no breaking change here. https://github.com/carbon-design-system/carbon-addons-iot-react/blame/9ae27a34377ea9cfe291773af897a4d306667b52/packages/react/src/components/DashboardEditor/__snapshots__/DashboardEditor.story.storyshot#L403